lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -159,6 +159,44 @@
     }
   }
 
+  /* Start: Override style for CCX grade book */
+  .container {
```

Comment:
@pdpinch this is class used by grade book. I am here fixing some css properties to make grade book work under div in side ccx dashboard. This class will be use by both instructor and coach. 

https://github.com/mitocw/edx-platform/pull/90/files#diff-9fb9f7eb54cc9531c7ed284e577e62a5R41